code/modules/power/apc.dm

```diff
@@ -373,7 +374,7 @@
 			update_state |= UPDATE_OPENED1
 		if(opened==2)
 			update_state |= UPDATE_OPENED2
-	else if(emagged || hacker || failure_timer)
+	else if(emagged || (hacker && prob(20)) || failure_timer || (hacker && hacker.system_override))

```

Comment:
How to annoy people with if clauses 101: by optimizing them:

`else if (emagged || failure_timer || (hacker && (hacker.system_override || prob(20)))`

Not mandatory, I'm just shitposting mainly. Though it is more optimal!